src/components/Chat/Chat.js

```diff
@@ -57,19 +71,41 @@ const Chat = ({ location }) => {
   const sendShoutout = (event) => {
     event.preventDefault();
     if (newShoutout) {
-      socket.emit("sendShoutout", {
-        ...newShoutout,
-        author: user,
-        id: uuidv4()
-      }, () => setNewShoutout(emptyShoutout));
+      socket.emit(
+        "sendShoutout",
+        {
+          ...newShoutout,
+          author: user,
+          id: uuidv4(),
+        },
+        () => setNewShoutout(emptyShoutout)
+      );
     }
   };
 
   return (
     <div className="outerContainer">
```

Comment:
This scroll solution works and I like the smoothness of it. The disadvantage is that the button disappears (though we could just make our own) and the scrollbar is put in front of the header and Canvas elements (also something we could fix). 

I am able to get your original approach with the 'react-scroll-to-bottom' package to work if I remove the outer container entirely. Here's what this return statement would look like:

```
     <div className="container">
        {shouldDropConfetti && <Confetti
          run={shouldDropConfetti}
          gravity={0.2}
          numberOfPieces={2000}
          width={width}
          height={height}
          recycle={false}
        />}
        <Header />
        <MessageList messageList={messageList} handleConfetti={toggleConfetti}/>
        <div className="bottomSection">
          <div id="canvasAndDogHouseContainer">
            <Canvas
              newShoutout={newShoutout}
              sendShoutout={sendShoutout}
              setNewShoutout={setNewShoutout}
            />
            <DogHouse />
          </div>
        </div>
      </div>
```